htdocs/fourn/commande/dispatch.php

```diff
@@ -465,11 +490,33 @@
 					$errors = $mouv->errors;
 					$error++;
 				} else {
-					$mouv->origin = &$object;
-					$result = $mouv->reception($user, $product, $supplierorderdispatch->fk_entrepot, $supplierorderdispatch->qty, $price, $comment, $eatby, $sellby, $batch);
-					if ($result < 0) {
-						$errors = $mouv->errors;
+					// if serial number management, delete before recreate (to avoid problem of already existing product_lot)
```

Comment:
I don't understand why deleting ProductLot is required.
A record is entered into the table the first time we see the lot. But then we are able to make other move using them lot with no need to delete the record.